docs/shared_parsers_catalog/neutron_server_log.rst

```diff
@@ -0,0 +1,9 @@
+Neutron Server Log
+==================
+
+NeutronServerLog - ``/var/log/neutron/server.log``
+--------------------------------------------------
+.. autoclass:: insights.parsers.neutron_server_log.NeutronServerLog
+   :members:
+   :show-inheritance:
+
```

Comment:
I sort of agree, especially since it means that new classes don't get automatically added to the documentaiton.  But I really want the consistency of presentation this gives us.  I'm not sure what a good solution looks like - we should talk about this in the documentation issue https://github.com/RedHatInsights/insights-core/issues/132 .